src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue28656.cs

```diff
@@ -0,0 +1,22 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+public class Issue28656 : _IssuesUITest
+{
+	public Issue28656(TestDevice testDevice) : base(testDevice)
+	{
+	}
+
+	public override string Issue => "CollectionView CollectionViewHandler2 does not change ItemsLayout on DataTrigger";
+
+	[Test]
+	[Category(UITestCategories.CollectionView)]
+	public void CollectionViewShouldChangeItemsLayout()
+	{
+		App.WaitForElement("ChangeLayoutButton");
+		App.Click("ChangeLayoutButton");
+		VerifyScreenshot();
```

Comment:
Pending snapshots already available in the latest build.

Example, Android:
![image](https://github.com/user-attachments/assets/c83e527b-2f91-4a43-bc3d-6f0918d37299)

Could you commit the images?